app/Jobs/CreateArticle.php

```diff
@@ -56,6 +58,10 @@ public function handle(): Article
         $article->authoredBy($this->author);
         $article->syncTags($this->tags);
 
+        if (App::environment() !== 'testing') {
```

Comment:
I don't think we should check the environment here. I think the event should be fired in all environments even if we don't want this particular listener to be called.

Reply:
I did deactivate this event on testing environment so it doesn't send annoying fake notification to Telegram channels. I have to check the env for this purpose here in `CreateArticle Job` or in `SendTelegramNewArticleNotification listener` what do you think!

Reply:
I'd rather not conditionally fire the event in case we want to hook other listeners in.

If we don't have the environment variables set in testing, does it throw an exception or just silently continue?

Reply:
I have just set Telegram related env variables to null in testing and it silently continue! 😄 I'll push now